pages/AWorkedExample.html

```diff
@@ -98,7 +98,7 @@ <h4>Second Card</h4>
     the Exploration Overview shows two cards.
   </p>
   <p>Add the following text:</p>
-  <img src="static/images/tutorial/card2Content.png" class="tutorial-image">
+  <img src="static/images/tutorial/card2Content.png" class="tutorial-image" alt="A text-area showing content for card 2, followed by a white 'cancel' button and a green 'save content' button at the bottom right of the text-area.">"
```

Comment:
Be more explicit about the textual content in this image. If you look at the surrounding text, you'll see that it says "Add the following text", but the alt text does not say what the text is.

In general, one way to check your alt texts might be to read this page as it looks in the Files Changed tab in GitHub, without looking at the pictures. I.e., imagine that, on the page, all the pictures were replaced by the alt text, or maybe a snippet like "An image showing {{whatever the alt text is}}". Does the whole thing read reasonably?